stdlib/stringLabels.mli

```diff
@@ -207,6 +207,18 @@ val split_on_char : sep:char -> string -> string list
 
     @since 4.04 (4.05 in StringLabels) *)
 
+val cut : sep:char -> string -> (string * string) option
```

Comment:
I'm wondering if this would be a good first function to use a labelled tuple in the result 🫣 (@ccasin, @goldfirere, @OlivierNicole - any JS examples, even potentially against doing this?) e.g. key/value, first/second, prefix/suffix, left/right?

If we were to go with a labelled tuple, I commit to sorting out #11792 (i.e. making syncing the docs work for this...)

Reply:
I think labeled tuples are a fine idea to consider whenever you have a tuple with multiple components of the same type. (Or not the same type, but less so.) That said, given that there is a pretty clear ordering involved in this particular case, I don't personally think labels are called for here.